prusti-tests/tests/verify/fail/quantifiers/triggers.rs

```diff
@@ -28,7 +28,7 @@ fn dummy(_n: isize, _res: isize) -> bool { true }
 
 #[requires(
     // This is an unsound quantifier to check whether triggerring works.
-    forall(|n: isize, res: isize| fib(n) == res, triggers=[(dummy(n, res),)])
+    forall(|n: isize, res: isize| fib(n) == res, triggers=[(dummy(n, res))])
```

Comment:
This change looks wrong: triggers are vectors of tuples. How it is recognized that `(dummy(n, res))` is a tuple?

Reply:
I'll undo the change and allow trailing-comma syntax. However, `triggers=[(foo)]` will also be recognised as the singular trigger set with `foo` as its one expression. This is not parsed by Rust or `syn`, so the comma is optional even for one-expression trigger sets.

Reply:
Sounds good.